FoundationBuilder/FoundationBuilder.vcxproj

```diff
@@ -100,6 +100,7 @@
       <PreprocessorDefinitions>_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ConformanceMode>true</ConformanceMode>
       <OpenMPSupport>true</OpenMPSupport>
+      <AdditionalIncludeDirectories>C:\Users\asus\source\repos\VladGribinchuk\FoundationBuilder\GeometryCore;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
This is still absolute path in FoundationBuilder project

Reply:
If this is not necessary change - you can simply open this file by notepad and just remove this line)